storage_delegation/main.tf

```diff
@@ -1,3 +1,10 @@
+module "crn_parser_kms_key" {
```

Comment:
Please rename it to `kms_key_crn_parser` to be consistent with all other modules.